src/string_piece.h

```diff
@@ -68,4 +68,51 @@ struct StringPiece {
   size_t len_;
 };
 
+#ifdef _WIN32
+// Same as StringPiece but handles wchar_t
+struct WStringPiece {
```

Comment:
@jlonnberg 

Much better would be to do the following:

Rename StringPiece to BasicStringPiece.

Make BasicStringPiece a template class.

Have StringPiece be a typedef of BasicStringPiece<char>
Have WStringPiece be a typedef of BasicStringPiece<wchar_t>